examples/cuda/riscv.mk

```diff
@@ -73,6 +79,7 @@ NC=\033[0m
 _REPO_ROOT ?= $(shell git rev-parse --show-toplevel)
 -include $(_REPO_ROOT)/environment.mk
 -include $(BSG_MACHINE_PATH)/Makefile.machine.include
+-include $(BSG_MANYCORE_DIR)/software/mk/Makefile.dimensions
```

Comment:
so we are controlling the tile group dimensions using a makefile in manycore?

Reply:
hmmmm. I think this is what happens when building non-parameterized apps? 
https://github.com/cornell-brg/bsg_manycore/blob/63255d758e3bf5c3a55ff5a891f983ac98d11d8a/software/mk/Makefile.builddefs#L17

Reply:
So if I get it right ...
https://github.com/cornell-brg/bsg_manycore/blob/63255d758e3bf5c3a55ff5a891f983ac98d11d8a/software/spmd/bsg_cuda_lite_runtime/appl-fib/Makefile#L27
which links
https://github.com/cornell-brg/bsg_manycore/blob/63255d758e3bf5c3a55ff5a891f983ac98d11d8a/software/spmd/Makefile.include#L11
which links
https://github.com/cornell-brg/bsg_manycore/blob/63255d758e3bf5c3a55ff5a891f983ac98d11d8a/software/mk/Makefile.master#L9
which links
https://github.com/cornell-brg/bsg_manycore/blob/63255d758e3bf5c3a55ff5a891f983ac98d11d8a/software/mk/Makefile.builddefs#L17